src/compiler/diagnosticMessages.json

```diff
@@ -2443,10 +2443,7 @@
         "category": "Error",
         "code": 2568
     },
-    "Type '{0}' is not an array type or a string type. Use compiler option '--downlevelIteration' to allow iterating of iterators.": {
-        "category": "Error",
-        "code": 2569
-    },
+

```

Comment:
Probably don't intend to leave an empty line here?

Reply:
Sometimes I do this so that a person can easily find the next available error code slot.

Reply:
AFAIK the documented rule is "largest unused code", so old codes are never reused anyway.

Reply:
> so old codes are never reused anyway

oh sweet summer child